kempner_computing_handbook/s4_neuro_ai_workflows/README.md

```diff
@@ -1,10 +1,12 @@
 # Spike Sorting
 
-Spike sorting is a crucial step in analyzing electrophysiological data from the brain. In collaboration with the Allen Institute for AI, the Kempner Research Engineering team developed these pipelines to support researchers on the FASRC and HMS RC computing clusters.
+Spike sorting is a crucial step in analyzing electrophysiological data from the brain. In collaboration with the [Allen Institute](https://alleninstitute.org/), the Kempner Research Engineering team developed these pipelines to support researchers on the [FASRC](https://www.rc.fas.harvard.edu/) and [HMS RC](https://it.hms.harvard.edu/about/departments/research-computing) computing clusters.
 
 Please refer Cluster specific documentation. 
```

Comment:
@dmbala, I think we need to also remove the last sentence, correct?

Reply:
Thanks @Naeemkh I agree, the last sentence is not needed. I made the changes.